### PR TITLE
Bug resolve

### PR DESCRIPTION
**Pull request by soni-muskan1234**
changed file generic.py and style.py
replaced if axis in [...] with get_axis_number(axis)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM quay.io/condaforge/miniforge3
 
 # if you forked pandas, you can pass in your own GitHub username to use your fork
 # i.e. gh_username=myname
-ARG gh_username=pandas-dev
+ARG gh_username=myname
```
Comment (phofl) [inline Dockerfile:5]: Please change back

Comment (soni-muskan1234) [reply Dockerfile:5]: will you please brief me about what to do now, as i am new to open source.
How to run precommit tests and what if got error in it while running doctests.

**Comment by alimcmaster1** [comment]
@soni-muskan1234 let us know if you want to carry on working on this. See comments from maintainers above on next steps.

**Comment by MarcoGorelli** [comment]
thanks @soni-muskan1234 - closing as the issue is now resolved

**Pull request closed without merging**